src/main/java/uk/gov/hmcts/sscs/service/PdfStoreService.java

```diff
@@ -0,0 +1,51 @@
+package uk.gov.hmcts.sscs.service;
+
+import static java.util.Collections.singletonList;
+import static org.springframework.http.MediaType.APPLICATION_PDF;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Collections;
+import java.util.List;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestClientException;
+import uk.gov.hmcts.reform.document.domain.UploadResponse;
+import uk.gov.hmcts.sscs.model.ccd.DocumentLink;
+import uk.gov.hmcts.sscs.model.ccd.SscsDocument;
+import uk.gov.hmcts.sscs.model.ccd.SscsDocumentDetails;
+import uk.gov.hmcts.sscs.service.evidence.EvidenceManagementService;
+
+@Service
+@Slf4j
+public class PdfStoreService {
+    private final EvidenceManagementService evidenceManagementService;
+
+    @Autowired
+    public PdfStoreService(EvidenceManagementService evidenceManagementService) {
+        this.evidenceManagementService = evidenceManagementService;
+    }
+
+    public List<SscsDocument> store(byte[] content, String fileName) {
+        ByteArrayMultipartFile file = new ByteArrayMultipartFile(content, fileName, APPLICATION_PDF);
+        try {
+            UploadResponse upload = evidenceManagementService.upload(singletonList(file));
+            String location = upload.getEmbedded().getDocuments().get(0).links.self.href;
+
+            DocumentLink documentLink = DocumentLink.builder().documentUrl(location).build();
+            SscsDocumentDetails sscsDocumentDetails = SscsDocumentDetails.builder()
+                    .documentFileName(fileName)
+                    .documentDateAdded(LocalDateTime.now().format(DateTimeFormatter.ISO_DATE))
+                    .documentLink(documentLink)
+                    .build();
+            SscsDocument pdfDocument = SscsDocument.builder().value(sscsDocumentDetails).build();
+
+            return Collections.singletonList(pdfDocument);
+        } catch (RestClientException exc) {
+            log.info("Failed to store pdf document but carrying on [" + fileName + "]");
+            log.debug("Exception from document store", exc);
```

Comment:
It will be useful if the exception is also logged at info level. Do we need two log level?